openapi.yaml

```diff
@@ -191,36 +197,68 @@ paths:
           required: false
           schema:
             type: string
+            example: John
         - in: query
           name: filter[lastName]
           description: Search by member last name.
           required: false
           schema:
             type: string
+            example: Doe
         - in: query
           name: filter[email]
           description: Search by member email.
           required: false
           schema:
             type: string
+            example: 'john.doe@example.com'
+        - in: query
+          name: filter[address][fuzzy]
+          description: Search by member street address.
+          required: false
+          schema:
+            type: string
+            example: 123 Main Street
+        - in: query
+          name: filter[city]
+          description: Search by the city of member.
+          required: false
+          schema:
+            type: string
+            example: Corvallis
+        - in: query
+          name: filter[state]
+          description: Search by the state of member.
+          required: false
+          schema:
+            type: string
+            pattern: '^[A-Z]{2}$'
+            example: OR
         - in: query
-          name: filter[address]
-          description: Search by member address.
+          name: filter[country]
+          description: Search by the country of member.
           required: false
           schema:
             type: string
+            example: United States of America
```

Comment:
Yeah, it could work, but we mostly use abbreviations (e.g. `filter[country]=US`) for country search and it will make the query easier.

Reply:
done.